src/Files.App/Views/Properties/LibraryPage.xaml.cs

```diff
@@ -107,7 +95,7 @@ private async Task AddLocationAsync()
 			if (!result)
 				return;
 
-			var folder = await StorageHelpers.ToStorageItem<StorageFile>(filePath);
+			var folder = await StorageHelpers.ToStorageItem<BaseStorageFolder>(filePath);
```

Comment:
This is another place that had the same problem.

Reply:
Yes. I saw this method with StorageFile was causing cast error in export/import settings command.
Should we basically use BaseStorageFolder until we fully migrate to Files.App.Storage?

Reply:
Yes, otherwise it wouldn't work.